dpdata/plugins/vasp.py

```diff
@@ -56,6 +56,11 @@ class VASPOutcarFormat(Format):
     @Format.post("rot_lower_triangular")
     def from_labeled_system(self, file_name, begin=0, step=1, **kwargs):
         data = {}
+        ml = False
+        if kwargs is not None:
+            for key, value in kwargs.items():
+                if key=="ml":
+                    ml = value
```

Comment:
These lines can be written as one line
```py
ml = kwargs.get("ml", False)
```
or it can be directly written in the `def` line

Reply:
change the code into one line